tools/jinja2-filters.py

```diff
@@ -0,0 +1,19 @@
+import base64
+
+class SecretIsNotString(Exception):
+    pass
+
+class NotSupportedDataType(Exception):
+    pass
+
+def filter_b64decode(secret_value, data_type = "string"):
+    if not isinstance(secret_value, str):
+        raise SecretIsNotString
```

Comment:
This is a generic base64 filter. It does not have to have anything to do with secrets. It's just how we use it. Your names will be very confusing if someone uses it for something else than encoding secrets. Please rename classes and variables to be more generic:
- `secret_value` -> `encoded_value`
- `SecretIsNotString` -> `NotAString`

Also rename `data_type` to `return_type`.